.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Set up Go 1.14
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Download dependencies
+        run: go mod download
+
+      - name: Run tests
+        run: go test -race ./...
+
```

Comment:
I think `run: make test` is all you need here. Have you tried that?

Reply:
Yes, if I do `make test` it [also works](https://github.com/mogaal/cloud-platform-cli/runs/903522232?check_suite_focus=true) but I can't see the value of depending on a makefile (which does `for`, `find`(s), etc) to run tests when the only thing you have to do is `go test -race ./...`

Reply:
Fair point.